cmd/pipeline/deploy.go

```diff
@@ -142,61 +150,70 @@ func (pc *Command) ExecuteDeployPipeline(ctx context.Context, opts *DeployOption
 		return fmt.Errorf("failed to load okteto context '%s': %v", okteto.Context().Name, err)
 	}
 
-	if opts.SkipIfExists {
-		cfgName := pipeline.TranslatePipelineName(opts.Name)
-
-		cfg, err := configmaps.Get(ctx, cfgName, opts.Namespace, c)
-		if err != nil && !oktetoErrors.IsNotFound(err) {
+	exists := false
+	cfgName := pipeline.TranslatePipelineName(opts.Name)
+	cfg, err := configmaps.Get(ctx, cfgName, opts.Namespace, c)
+	if err != nil {
+		if opts.ReuseParams && oktetoErrors.IsNotFound(err) {
+			return errUnableToReuseParams
+		}
+		if opts.SkipIfExists && !oktetoErrors.IsNotFound(err) {
 			return fmt.Errorf("failed to get pipeline '%s': %w", cfgName, err)
 		}
-		if err == nil {
-			if cfg != nil && cfg.Data != nil {
-				if cfg.Data["status"] == pipeline.DeployedStatus {
-					oktetoLog.Success("Skipping repository '%s' because it's already deployed", opts.Name)
-					return nil
-				}
+	}
+	exists = cfg != nil && cfg.Data != nil
 
-				if !opts.Wait && cfg.Data["status"] == pipeline.ProgressingStatus {
-					oktetoLog.Success("Repository '%s' already scheduled for deployment", opts.Name)
-					return nil
-				}
+	if opts.ReuseParams && exists {
+		if overrideOptions := cfgToDeployOptions(cfg); overrideOptions != nil {
+			applyOverrideToOptions(opts, overrideOptions)
```

Comment:
```suggestion
			opts = overrideOptions
```
can we do it like this suggestion?

Reply:
i did it with the function to select which ones to override. There are other options set before this that could be changed to empty if i assign the value directly...